aspnetcore/tutorials/razor-pages/razor-pages-start/sample/RazorPagesMovie30/Startup.cs

```diff
@@ -33,7 +33,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddRazorPages();
 
             services.AddDbContext<RazorPagesMovieContext>(options =>
-                options.UseSqlite(Configuration.GetConnectionString("RazorPagesMovieContext")));
+                options.UseSqlite(Configuration.GetConnectionString("MovieContext")));
```

Comment:
This might be the right choice, but you should consider changing the `appsettings.json` file for Sqlite to use `RazorPagesMovieContext"`

Reply:
Yes, that is the better fix. Going with "RazorPagesMovieContext" in appsettings.json. It keeps it consistent with VS version.  Although the longer.  The longer the project name the more unwieldly these become, but it is the right choice.